.github/workflows/npm_publish.yml

```diff
@@ -0,0 +1,30 @@
+name: npm publish
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
On some of our other projects we perform the publish when the release is created on GitHub. The advantage is someone accidentally pushing a ton of tags won't generate a ton of publishes. Also GitHub releases alert watches of the repo which is a nice way for people to keep track. See this example for how to set that up:
https://github.com/stellar/js-stellar-sdk/blob/9926203899a69296febdd4172d9f19438f88d239/.github/workflows/npm_publish.yml#L2-L4